node/src/Commands.ts

```diff
@@ -1850,3 +1850,33 @@ export function createZRevRankWithScore(
 ): command_request.Command {
     return createCommand(RequestType.ZRevRank, [key, member, "WITHSCORE"]);
 }
+
+/**
+ * Mandatory option for zmpop.
+ * Defines which elements to pop from the sorted set.
+ */
+export enum ScoreFilter {
```

Comment:
Should we put it into `src/commands` dir?

Reply:
Can you add the two necessary occurrences for ScoreFilter in npm/glide/index.ts?

Reply:
> Should we put it into src/commands dir?
This has yet to be discussed if we separate enums and option types in to separate files.